2018-03-28/azblob/zt_examples_test.go

```diff
@@ -971,7 +973,8 @@ func ExampleUploadFileToBlockBlobAndDownloadItBack() {
 
 	// Create a BlockBlobURL object to a blob in the container (we assume the container already exists).
 	u, _ := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net/mycontainer/BigBlockBlob.bin", accountName))
-	blockBlobURL := azblob.NewBlockBlobURL(*u, azblob.NewPipeline(azblob.NewSharedKeyCredential(accountName, accountKey), azblob.PipelineOptions{}))
+	credential, _ := azblob.NewSharedKeyCredential(accountName, accountKey)
+	blockBlobURL := azblob.NewBlockBlobURL(*u, azblob.NewPipeline(credential, azblob.PipelineOptions{}))
```

Comment:
given this is an example (and thus a recommended way of using this library) - shouldn't this be checking that the error object returned is nil?

in addition - if `u` is nil here (as the error isn't caught above) there's a panic here?